drupal/sites/default/boinc/modules/boincuser/boincuser.module

```diff
@@ -1205,8 +1205,8 @@ function join_page($type = null) {
    * here. An improvement would be admin settings for the Join Page
    * where this path could be set.
    */
-  $ruleslinkA = '/rules-and-policies';
-  $ruleslinkB = '/content/rules-and-policies';
+  $ruleslinkA = $base_url . '/rules-and-policies';
```

Comment:
This actually breaks the menu_valid_path() functionality, which always returns TRUE for absolute URLs. If $base_url is needed, it should be added just prior to rendering the link (though the l() function should handle this ok, too).